src/LaunchDarkly.Tests/LdClientManualTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using LaunchDarkly.Client;
+using NUnit.Framework;
+using System.Threading.Tasks;
+
+namespace LaunchDarkly.Tests
+{
+    class LdClientManualTest
+    {
+        private static string API_KEY = "sdk-707fa2a8-f3be-4f14-a122-946ab580a648";
```

Comment:
What is this SDK key for? Suggest resetting it if you accidentally checked it in.